lib/linter/apply-disable-directives.js

```diff
@@ -201,7 +201,6 @@ function processUnusedDisableDirectives(allDirectives) {
  */
```

Comment:
Please check the JSDoc comments for this function. I believe this PR changes how the function works and so the comments should be updated to reflect the new behavior.